lib/test/t_IterativeMoments_std.cxx

```diff
@@ -0,0 +1,126 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The test file of class IterativeMoments for standard methods
+ *
+ *  Copyright 2005-2019 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include "openturns/OT.hxx"
+#include "openturns/OTtestcode.hxx"
+
+using namespace OT;
+using namespace OT::Test;
+
+// Check if bool is true
+void assertTrue(bool condition)
```

Comment:
That function should be removed and replaced with assert_equal. The corresponding commit is at: e99bf00622873390f92fc9763171ae25d537ae5b